documentation/sequence diagrams/event/Create Event sequence.txt

```diff
@@ -0,0 +1,37 @@
+title Create Event
+participant User
+participant UI EventList
+participant UI CreateEvent Dialog
+participant EventView
+participant HttpRequest
+participant Event
+
+User->+UI EventList: Click button Create
+UI EventList->+UI CreateEvent Dialog: open CreateEvent Dialog
+User->UI CreateEvent Dialog: Enter data for creating a new event
+User->UI CreateEvent Dialog: Click button Submit
+alt not success
+UI CreateEvent Dialog->+EventView: post(request: HttpRequest, team_id: int)
```

Comment:
I think, post request goes before success or not success statement

Reply:
Done!